src/app/pages/StoryPage/index.jsx

```diff
@@ -84,6 +82,9 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
     pageData,
   );
   const featuresInitialData = path(['secondaryColumn', 'features'], pageData);
+  const recommendationsIntialData = path(['recommendations'], pageData);
+
+  console.log(recommendationsIntialData);
```

Comment:
Currently console logging the data here to show it's passing the data correctly, as the front-end container isn't consuming the data properly. This will behave correctly once the work in https://github.com/bbc/simorgh/pull/7225 has been completed. This will be removed before this is merged.